docs/usage-guide/topics/ch09-certificates.md

```diff
@@ -29,12 +29,30 @@ default system certificates to these configs, call `s2n_config_load_system_certs
 
 ## Server Authentication
 
-A server must have a certificate and private key pair to prove its identity. s2n-tls supports RSA, RSA-PSS, and ECDSA certificates, and allows one of each type to be added to a config.
+A server must have a certificate and private key pair to prove its identity. s2n-tls supports RSA, RSA-PSS, and ECDSA certificates, and allows one of each type to be added to a config for a given domain name.
 
 Create a new certificate and key pair by calling `s2n_cert_chain_and_key_new()`, then load the pem-encoded data with `s2n_cert_chain_and_key_load_pem_bytes()`.  Call `s2n_config_add_cert_chain_and_key_to_store()` to add the certificate and key pair to the config. When a certificate and key pair is no longer needed, it must be cleaned up with `s2n_cert_chain_and_key_free()`.
 
 A client can add restrictions on the certificate’s hostname by setting a custom `s2n_verify_host_fn` with `s2n_config_set_verify_host_callback()` or `s2n_connection_set_verify_host_callback()`. The default behavior is to require that the hostname match the server name set with `s2n_set_server_name()`.
 
+### SNI
+
+TLS servers will often serve multiple domains from a single IP address, with each domain potentially requiring its own certificate. When a TLS client receives a server's certificate, it will check to ensure that it was issued for the domain that it's connecting to. As such, the server needs to know which domain the client is connecting to in order to send the correct certificate. This information is communicated to the server via the Server Name Indication (SNI) extension.
+
+#### Client configuration
+
+`s2n_set_server_name()` is used on client connections to specify a domain name in the SNI extension.
+
+#### Server configuration
+
+Certificates loaded with `s2n_config_add_cert_chain_and_key_to_store()` are automatically associated with each domain indicated in the certificate's Subject Alternative Name (SAN) field. If a loaded certificate doesn't contain a SAN field, the Common Name (CN) field is used instead. While negotiating a TLS connection, s2n-tls servers will automatically send clients a loaded certificate that matches the value of the client's SNI extension.
```

Comment:
Can you somehow call out more clearly that you can have multiple domains in the SAN field? Right now it's a little vague.

Reply:
Also this is a total nit but some of your sentences could go from passive --> active. e.g. "The Common Name (CN) field is used instead if a loaded certificate doesn't contain a SAN field."

Reply:
I tried to update everything to be active but I'm not good at distinguishing between active and passive, so if I missed something let me know.